source/fundamentals/crud/read-operations/text.txt

```diff
@@ -286,56 +300,59 @@ You can also include the ``$text`` evaluation query operator in the
 :manual:`$match </reference/operator/aggregation/match/>` stage to
 perform a text search in an aggregation pipeline.
 
-Term Search Example
+Match a Search Term
 ~~~~~~~~~~~~~~~~~~~
 
-The following example runs a text search for titles that contain the term "Winter":
+The following example runs a text search for descriptions that contain the term "herb":
 
 .. io-code-block::
    :copyable: true
 
    .. input::
       :language: go
 
-      matchStage := bson.D{{"$match", bson.D{{"$text", bson.D{{"$search", "Winter"}}}}}}
+      matchStage := bson.D{{"$match", bson.D{{"$text", bson.D{{"$search", "herb"}}}}}}
 
       cursor, err := coll.Aggregate(context.TODO(), mongo.Pipeline{matchStage})
       if err != nil {
          panic(err)
       }
 
-      var results []bson.D
+      var results []Dish
       if err = cursor.All(context.TODO(), &results); err != nil {
          panic(err)
       }
+
       for _, result := range results {
-         fmt.Println(result)
+          res, _ := json.Marshal(result)
+          fmt.Println(string(res))
       }
 
    .. output::
       :language: none
       :visible: false
 
-      [{_id ObjectID("...")} {title Captain America: The Winter Soldier} {year 2014}]
+      {"Name":"Kale Tabbouleh","Description":"A bright, herb-based salad. A perfect starter for vegetarians and vegans."}
+      {"Name":"Herbed Whole Branzino","Description":"Grilled whole fish stuffed with herbs and pomegranate seeds. Serves 3-4."}
 
-Text Score Example
-~~~~~~~~~~~~~~~~~~
+Match then Sort by Relevance
```

Comment:
Issue: The word "then" sticks out to me here. It feels like it should have some punctuation, which could look strange in a heading.

Suggestion: 

```suggestion
Match and Sort by Relevance
```